testing/scenario_app/lib/main.dart

```diff
@@ -53,32 +22,41 @@ void main() {
     ..onMetricsChanged = _onMetricsChanged
     ..onPointerDataPacket = _onPointerDataPacket
     ..scheduleFrame();
+
   final ByteData data = ByteData(1);
   data.setUint8(0, 1);
   window.sendPlatformMessage('waiting_for_status', data, null);
 }
 
+void _handleDriverMessage(Map<String, dynamic> call) {
+  final String methodName = call['method'] as String;
+  switch (methodName) {
+    case 'set_scenario':
+      assert(call['args'] != null);
+      loadScenario(call['args'] as Map<String, dynamic>);
+      final String scenarioName = scenarioData['name'] as String;
+      print('Loading scenario $scenarioName');
+    break;
+    default:
+      print('Unimplemented method: $methodName.');
```

Comment:
Github ate my review comment - I think this should throw.

Reply:
Done.